### PR TITLE
more cleanups

### PR DESCRIPTION
**Pull request by tarekziade**
Silence!

### CODE REVIEW

File: connectors/sources/tests/test_gcs.py [hunk 1]
```diff
@@ -72,7 +72,7 @@ async def test_empty_configuration():
 
 
 @pytest.mark.asyncio
-async def test_ping_for_successful_connection():
+async def test_ping_for_successful_connection(catch_stdout, patch_logger):
```
Comment (navarone-feekery) [inline connectors/sources/tests/test_gcs.py:75]: So I'm sure this is fine, but I'm curious; where do these arguments come from? Is this just part of the testing lib or something?

**Review by navarone-feekery — APPROVED** [review]
LGTM with a comment

File: connectors/sources/tests/test_gcs.py (hunk 1 above)
Comment (tarekziade) [reply connectors/sources/tests/test_gcs.py:75]: yes, this is part of `pytest` fixtures.

every function decorated with `@pytest.fixture` in https://github.com/elastic/connectors-python/blob/main/connectors/conftest.py can be used in tests as arguments like this.

when they are added, the fixture is used as a wrapper :

```
def fixture():
    # do something before the test
    yield  # this calls the tests
    # do something after the test
```

Comment (tarekziade) [reply connectors/sources/tests/test_gcs.py:75]: more on fixtures https://docs.pytest.org/en/6.2.x/fixture.html

**Comment by tarekziade** [comment]
The backport was not needed (GCS is 8.7 only)

**Pull request merged**